prost/src/lib.rs

```diff
@@ -1,6 +1,10 @@
 #![doc(html_root_url = "https://docs.rs/prost/0.12.2")]
-#![cfg_attr(not(feature = "std"), no_std)]
 #![doc = include_str!("../../README.md")]
+#![no_std]
+
+// See: https://github.com/tokio-rs/prost/pull/1047
```

Comment:
The code should not refer to Github issues, but instead contain the contents directly